setup/install/providers/kubernetes-v2/gke/index.md

```diff
@@ -0,0 +1,62 @@
+---
+Layout: single
+title:  "Google Kubernetes Engine (GKE) Setup"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+This page describes how to set up a Kubernetes cluster on
+[GKE](https://cloud.google.com/kubernetes-engine/) to be used as a Spinnaker
+provider. The process is very simple, but you need to do some specific things
```

Comment:
Is it worth noting that it's a v2 provider? It'll be obvious if the user gets here from the nav, but otherwise, maybe not?

Reply:
Will call it out just in case